fastestimator/schedule/lr_shedule.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import math
+
+
+def cosine_decay(epoch_or_step: int, cycle_length: int, init_lr: float, min_lr: float = 1e-6, start: int = 0):
```

Comment:
maybe just 'time' instead of 'epoch_or_step'

Reply:
done